share/doc/homebrew/Formula-Cookbook.md

```diff
@@ -70,7 +70,7 @@ so you can override this with `brew create <url> --set-name <name>`.
 
 ## Fill in the [`homepage`](http://www.rubydoc.info/github/Homebrew/homebrew/master/Formula#homepage%3D-class_method)
 
-**We don’t accept formulae without [`homepage`](http://www.rubydoc.info/github/Homebrew/homebrew/master/Formula#homepage%3D-class_method)s!**
+**We don’t accept formulas without [`homepages`](http://www.rubydoc.info/github/Homebrew/homebrew/master/Formula#homepage%3D-class_method)!**
```

Comment:
I can actually restore the formulae, but there was a missing s on the "home pages" link ...

Sent from my iPhone

> On Jan 22, 2016, at 06:53, Dominyk Tiller <notifications@github.com> wrote:
> 
> In share/doc/homebrew/Formula-Cookbook.md:
> 
> > @@ -70,7 +70,7 @@ so you can override this with `brew create <url> --set-name <name>`.
> >  
> >  ## Fill in the [`homepage`](http://www.rubydoc.info/github/Homebrew/homebrew/master/Formula#homepage%3D-class_method)
> >  
> > -**We don’t accept formulae without [`homepage`](http://www.rubydoc.info/github/Homebrew/homebrew/master/Formula#homepage%3D-class_method)s!**
> > +**We don’t accept formulas without [`homepages`](http://www.rubydoc.info/github/Homebrew/homebrew/master/Formula#homepage%3D-class_method)!**
> formulae is actually correct here. It's the debate that will never die, but "formulae" is a valid plural of formula.
> 
> —
> Reply to this email directly or view it on GitHub.
>